tests/k8s_test.go

```diff
@@ -110,3 +118,7 @@ func TestK8sNetworkPolicyNode(t *testing.T) {
 func TestK8sNodeNode(t *testing.T) {
 	testNodeCreation(t, nil, nil, gremlin.NewValueString("node").Quote(), gremlin.NewValueString(nodeName).Quote())
 }
+
+func TestK8sNamespaceNode(t *testing.T) {
+	testNodeCreation(t, setupNamespace, tearDownNamespace, gremlin.NewValueString("namespace").Quote(), gremlin.NewValueString(namespaceName).Quote())
```

Comment:
I think it is time to add test to check that the links are created properly. By the way it is a bit hard to read what the test is really checking with this level of indirections. The query builder doesn't help. I think it is more convenient to have a complete gremlin expression readable. fmt.Sprintf(`G.V().Has("Manager", "%s", "Type": "%s", "Name": "%s")`).

Reply:
Will take this comment into account when extending the k8s tests (this effort can now continue after accepting the first k8s tests PR). In any case will be handled separately to this PR

Reply:
ok thx